src/pydata_sphinx_theme/assets/scripts/pydata-sphinx-theme.js

```diff
@@ -346,8 +346,8 @@ var setupSearchButtons = () => {
  */
 var getCurrentUrlPath = () => {
   if (DOCUMENTATION_OPTIONS.BUILDER == "dirhtml") {
-    return DOCUMENTATION_OPTIONS.pagename == "index"
-      ? `/`
+    return DOCUMENTATION_OPTIONS.pagename.endsWith("index")
+      ? `${DOCUMENTATION_OPTIONS.pagename.substring(0, DOCUMENTATION_OPTIONS.pagename.length - 5)}`
```

Comment:
I'm guessing `-5` is `len('.html')` ?
```suggestion
      ? `${DOCUMENTATION_OPTIONS.pagename.substring(0, DOCUMENTATION_OPTIONS.pagename.length - '.html'.length)}`
```

Maybe ?

Reply:
> I'm guessing `-5` is `len('.html')` ?

No, it removes trailing `index`, `len('index') == 5`